### ISSUE TITLE
Youtube Regrets Extension landing page

### ISSUE THREAD

**Issue by phildexter** [mozilla/foundation.mozilla.org#5081]
Brandi is the main stakeholder.

We are expecting some similar effects to the previous youtube page, with some scrolling at the top I believe.

Designs are here: https://lindsay970727.invisionapp.com/console/share/T41BYHZU8Y/479637009

**Comment by mmmavis** [comment]
Considering implementation for various screen sizes and custom stylings, I'm setting the estimate to `8`.

**Comment by mmmavis** [comment]
@phildexter do you know who will be providing copy and image files?

**Comment by cadecairos** [comment]
@mmmavis I got a request to wire up the extension download button with a GA event so we can count downloads. Let's take care of that as a stretch task.

**Comment by mmmavis** [comment]
Blocked until copy and image assets are provided.

**Comment by phildexter** [comment]
I've messaged Brandi about talking with you directly on this one @mmmavis  - worth you reaching out to her if you are still blocked on this after hompage work is done

**Comment by mmmavis** [comment]
I have copy from Brandi now but no image assets were provided. I'm gonna try exporting images from the Sketch file first and go from there.

**Comment by TheoChevalier** [comment]
After chatting with Brandi, we felt it was important to mention on the page that only videos in English will be included in the analysis, because even if we’re only actively promoting it in English, many non-English speakers will end up on the page regardless and we think it’s fair they are aware of this limitation.

So I’ve added a note on the page, but it’s only showing up for people visiting the localized pages, nothing changes on the /en/ page. You can see it on the French page for instance, after the 3 steps: https://foundation-s-issue-5081-l7ujfh.herokuapp.com/fr/campaigns/regrets-reporter/

I’ve pushed my changes to [this branch](https://github.com/mozilla/foundation.mozilla.org/tree/issue-5081-youtube-regrets-extension), hopefully that’s okay and won’t create conflicts.

Here’s a changelog of what I’ve done:

- Excluding the page template from localization. It’s not worth actually translating the page, since the extension only works with English-language videos, but if we ever need to translate it, we just have to revert the change in tasks.py. Making it ready for localization was definitely the right thing to do, we will just not translate it for now.
- Added a note only showing up on the English page (using `{% if request.LANGUAGE_CODE == "en" %}` to determine this)
- While I was there, fixed a couple typos in the copy (`What does *at* YouTube rabbit hole look like`, and another string referring to MozFest 2020 instead of MozFest 2019)
- Removed `/en` from links redirecting to the foundation or MozFest sites
- Simplified some of the string wrapping to remove markup from strings where possible (The less markup ends up in the string sent for localization, the less chances someone messes with it while translating)
- Made the dates of the timeline localizable (variables passed to {% blocktrans %} are not part of the localizable string, so if we move the date to a `{{date}}` variable, we’re actually hardcoding it)
- Wrapped an accessibility string for localization

CC @mmmavis
